src/TensorFlowNET.Core/NumPy/NDArray.Implicit.cs

```diff
@@ -14,7 +14,76 @@ public void Deconstruct(out byte blue, out byte green, out byte red)
             red = data[2];
         }
 
-        public static implicit operator NDArray(Array array)
+        public static implicit operator NDArray(int[] array)
```

Comment:
Is it possible to use like `public static implicit operator NDArray<T>(T[] array)`?

Reply:
It's attractive but C# does not support using generic implicit operators.🤣